### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by vivre111**

### CODE REVIEW

File: test.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+.item-a {
```
Comment (alfredchanfq) [inline test.html:5]: By the way, use "prettier" to format your codes.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//import { readFileSync } from 'fs';
+// input
+const handlebarsFileName = `temp.handlebars`;
+const data = {
+  input: {
+      isEndorsement: true
+  }
+}
+const template = handlebars.compile(
+  fs.readFileSync(
```
Comment (alfredchanfq) [inline main.js:13]: We use webpack loader to read the file. You'll know what that means once you put your work onto the document backend.

File: test.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  padding: 5px 5px;
+  border: 1px solid black;
+  background-color:white;
+  color: black;
+  text-transform: uppercase;
+}
+.grid-container > div.item-g{
+  text-align: center;
+}
+div[class^='item-'] > div{
```
Comment (alfredchanfq) [inline test.html:88]: I feel like you might want to declare a container class

```html
<div class="abc-container" >
  <div>...</div>
</div>
```

Then in the css do
```css
.abc-container > div {
  ...
}
```

**Review by HenryFeng-Foxquilt — APPROVED** [review]

File: test.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+<body>
+<div style="text-align: center; font-weight: bold; font-size: x-small; text-transform: uppercase;">
+  this endorsement changes the policy. please read it carefully
+</div>
+<div style="text-align: center; font-weight: bold; font-size: medium; padding:10px 100px">POLICY CHANGES</div>
+<div style="text-align:right;padding:10px 100px">
+  Policy Change Number
+</div>
+<div class="grid-container">
+  <div class="item-a">policy number</div>
```
Comment (alfredchanfq) [inline test.html:116]: Give a more meaningful name, like
```html
  <div class="policy-number">policy number</div>
```

**Review by alfredchanfq — APPROVED** [review]

File: test.html (hunk 4 above)
Comment (HenryFeng-Foxquilt) [inline test.html:116]: I think you could use `policy_num` here and use `item-a` for grid-area

File: test.html (hunk 1 above)
Comment (vivre111) [reply test.html:5]: accepted.

File: test.html (hunk 3 above)
Comment (vivre111) [reply test.html:88]: accepted

File: test.html (hunk 4 above)
Comment (vivre111) [reply test.html:116]: accepted

Comment (vivre111) [reply test.html:116]: accepted

**Pull request merged**